### ISSUE TITLE
Github login not working with SSL enabled

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-online-tools#221]
When using SSL, the following error occurs when trying to login using Github:

```
Request Method: | GET
-- | --
http://52.32.53.255/oauth/complete/github/?error=redirect_uri_mismatch&error_description=The+redirect_uri+MUST+match+the+registered+callback+URL+for+this+application.&error_uri=https%3A%2F%2Fdocs.github.com%2Fapps%2Fmanaging-oauth-apps%2Ftroubleshooting-authorization-request-errors%2F%23redirect-uri-mismatch&state=nnRwRg6fsZBXQkLSjTmk5maz6yePnA0V
1.11.29
AuthFailed
Authentication failed: The redirect_uri MUST match the registered callback URL for this application.
/usr/local/lib/python2.7/site-packages/social_core/backends/oauth.py in process_error, line 380
/usr/local/bin/python

...
```

**Comment by goneall** [comment]
The Github Oauth configuration will need to be updated to use https:// and the domain tools.spdx.org once we go live.  I'll make that change just before going into production with the new code.

**Comment by goneall** [comment]
Resolved in PR #258 - however there is now a new issue #260
